autopts/pybtp/btp/gap.py

```diff
@@ -112,6 +112,25 @@
                                   defs.GAP_SET_EXTENDED_ADVERTISING, CONTROLLER_INDEX, 1),
     "set_extend_advertising_off": (defs.BTP_SERVICE_ID_GAP,
                                    defs.GAP_SET_EXTENDED_ADVERTISING, CONTROLLER_INDEX, 0),
+    "padv_configure": (defs.BTP_SERVICE_ID_GAP,
+                       defs.GAP_PADV_CONFIGURE, CONTROLLER_INDEX),
+    "padv_start": (defs.BTP_SERVICE_ID_GAP,
+                   defs.GAP_PADV_START, CONTROLLER_INDEX, ""),
```

Comment:
One more thing to fix TypeError: BTPWorker.send() got multiple values for argument 'data'
```suggestion
                   defs.GAP_PADV_START, CONTROLLER_INDEX),
```